### PR TITLE
HTTP-119 use default certs when no security configured for https

### PR DESCRIPTION
**Pull request by davidradl**
#### Description

use default certs when no security configured for https

Resolves `HTTP119`

##### PR Checklist
- [x] Tests added
- [x]  [Changelog](CHANGELOG.md) updated

### CODE REVIEW

**Comment by davidradl** [comment]
@grzegorz8 @kristoffSC Please could you have a look at this one? thanks.

File: src/test/java/com/getindata/connectors/http/internal/table/lookup/querycreators/ElasticSearchLiteQueryCreatorTest.java [hunk 1]
```diff
@@ -18,6 +18,24 @@
 
 public class ElasticSearchLiteQueryCreatorTest {
 
+    @Test
```
Comment (grzegorz8) [inline src/test/java/com/getindata/connectors/http/internal/table/lookup/querycreators/ElasticSearchLiteQueryCreatorTest.java:21]: I think this PR should not contain irrelevant changes, so I added these tests in PR: https://github.com/getindata/flink-http-connector/pull/130
Please rebase your branch with main :)

**Review by grzegorz8 — CHANGES_REQUESTED** [review]

File: src/test/java/com/getindata/connectors/http/internal/table/lookup/querycreators/ElasticSearchLiteQueryCreatorTest.java (hunk 1 above)
Comment (davidradl) [reply src/test/java/com/getindata/connectors/http/internal/table/lookup/querycreators/ElasticSearchLiteQueryCreatorTest.java:21]: @grzegorz8 no worries - I was just pulling in the fix as is from Kristof as a base.

**Comment by davidradl** [comment]
@grzegorz8 I have made the changes - I ended up coding this fresh rather than rebasing - I hope this is ok.

**Review by grzegorz8 — APPROVED** [review]

**Comment by davidradl** [comment]
@grzegorz8 thanks for merging :-) . Would it be possible to put out a release on this code level?

**Pull request merged**